sdk/src/main/java/com/microsoft/hydralab/performance/PerformanceTestSpec.java

```diff
@@ -0,0 +1,61 @@
+package com.microsoft.hydralab.performance;
+
+import java.io.File;
+import java.util.UUID;
+
+public class PerformanceTestSpec {
+    public static final String INSPECTOR_ANDROID_MEMORY_DUMP = "AndroidMemoryDump";
+    public static final String INSPECTOR_ANDROID_MEMORY_INFO = "AndroidMemoryInfo";
+    public static final String INSPECTOR_ANDROID_BATTERY_INFO = "AndroidBatteryInfo";
+    public static final String INSPECTOR_WIN_BATTERY = "WindowsBattery";
+    public static final String INSPECTOR_WIN_MEMORY = "WindowsMemory";
+    public String id = UUID.randomUUID().toString();
+    public final String inspector;
+    public final String appId;
+    public final String deviceId;
+    public String name;
+    public final String inspectionKey;
+    /**
+     * TODO: where to set a value for this field?
+     */
+    public File resultFolder;
+    public long interval;
+
+    public PerformanceTestSpec(String inspector, String appId, String deviceId, String name) {
+        this.inspector = inspector;
+        this.appId = appId;
+        this.deviceId = deviceId;
+        this.name = name;
+        inspectionKey = String.format("%s-%s-%s", appId, deviceId, inspector);
```

Comment:
Do we support simultaneous performance testing on different android devices? If the device id is "Android", we may not tell them apart.